### DISCUSSION TITLE
Remove the /models subfolder ?

### DISCUSSION THREAD

**Discussion by blefaudeux** [facebookresearch/xformers#65] [Q&A] [answered]
Not really used, meant to be a way to show that you could build models yourself by picking parts one by one, but it's pretty obvious to anybody in the field I guess. I'm afraid that it could be confusing to people arriving in the repo

https://github.com/facebookresearch/xformers/tree/main/xformers/models

cc @SeanNaren @tchaton @dianaml0 @jieru-hu @Vittorio-Caggiano @iyerr3

**Comment by SeanNaren** [comment, marked as answer]
Hey @blefaudeux! apologies on the late response on this. I agree, I think the components are most important, and if someone want's to build something like this we should ask them to look at the factory/build it themselves :)

I think it gives the wrong impression that the library will scale out with specific model implementations which I do not think is the correct purpose?

**Comment by blefaudeux** [comment]
done, closing this
